about.html

```diff
@@ -110,7 +110,7 @@
             <div class="row">
                 <div class="col-12">
                     <nav class="navbar navbar-expand-lg navbar-custom">
-                        <a class="navbar-brand" href="#">
+                        <a class="navbar-brand" href="index.html">
                             <img class="logo-custom" src="./assets/image/logo.png" alt="" srcset="">
```

Comment:
- [ ] On the About Page the first section isn't styled as seen in the original design. 
- [ ] In the given design, the `main section` has this text `Aqurds Web Development Course 2023` properly centered on the page and has a smaller font size compared to your current design. I will suggest you incorporate this into your design looks similar to the original design 😉
- Your Design
<img width="1244" alt="Screen Shot 2023-05-24 at 11 01 01 AM" src="https://github.com/Aqurds/wd-course/assets/102699958/70ea296f-7359-4e86-aab0-b5ce6a166b0d">

- The Behance Design
<img width="692" alt="Screen Shot 2023-05-24 at 11 01 26 AM" src="https://github.com/Aqurds/wd-course/assets/102699958/2a5833db-8360-428b-b8bb-ff8de1eb9917">

- [ ] I also noticed that you're currently missing the underlined email text underneath this text `Please contact us at support@aqurds.com for more details` as seen in the behance design. So please ensure that there is similar text that is styled as seen in the Design. 👍
- Your Design
<img width="568" alt="Screen Shot 2023-05-24 at 11 02 11 AM" src="https://github.com/Aqurds/wd-course/assets/102699958/38961ff5-e064-4a63-ac92-54018390ad9e">

- The Behance Design
<img width="512" alt="Screen Shot 2023-05-24 at 11 02 31 AM" src="https://github.com/Aqurds/wd-course/assets/102699958/bcebef71-2911-4035-ae38-e4203cd9ddea">

- [ ] I also noticed that you're currently missing the middle section with a centered column that includes a section header, a divider, a paragraph of text, and an image or large logo. I have included an image of the Behnace Design Below 😉 
- The behance Design
<img width="742" alt="Screen Shot 2023-05-24 at 11 07 18 AM" src="https://github.com/Aqurds/wd-course/assets/102699958/6dd49809-8e2c-4160-ba7b-baaaba8d930c">

- [ ] Great job adding the third section but from the behance design, the images had some text centered over them. which you're currently missing in your design so kindly implement this as well 👍 
- Your Design
<img width="1267" alt="Screen Shot 2023-05-24 at 11 08 54 AM" src="https://github.com/Aqurds/wd-course/assets/102699958/fbe80c28-5cc2-41aa-b42a-d0dc94c36429">
- The Behance Design
<img width="594" alt="Screen Shot 2023-05-24 at 11 12 17 AM" src="https://github.com/Aqurds/wd-course/assets/102699958/948f2136-1e59-45ee-9df6-c753c6b3f5db">

- [ ] I also noticed that the footer of your About Page is white instead of black as seen in the original design so kindly implement this as well 😉 
- Your Design
<img width="1280" alt="Screen Shot 2023-05-24 at 11 12 53 AM" src="https://github.com/Aqurds/wd-course/assets/102699958/d04abba5-6642-4eb1-8034-78e946f8708b">

- The Behance Design
<img width="743" alt="Screen Shot 2023-05-24 at 11 14 48 AM" src="https://github.com/Aqurds/wd-course/assets/102699958/f6f1db5f-0b83-4a15-bd21-02507eef99fb">